src/erlfdb.erl

```diff
@@ -106,6 +106,7 @@
     % Transaction status
     get_next_tx_id/1,
     is_read_only/1,
+    get_approximate_transaction_size/1,
```

Comment:
get_approximate_transaction_size(Transaction)` seems a bit redundant. How about just `get_approximate_size`? Also this should go up a level. get_next_tx_id/is_read_only are both erlfdb specific and it could be misleading to lump this function with them. I'd just add a new section with `% Transaction size info` or similar.